README.md

```diff
@@ -253,6 +265,8 @@ The Symbolication Server includes its own tests. There are two options for
 running tests. `python runTests.py` runs all tests.
 `python tests/test_[name].py` runs just one test.
 
+If you have docker, you can run all the tests using `make test`.
+
```

Comment:
This does not seem to work. Every single test fails with the error: `OSError: [Errno 13] Permission denied: '/app/pids'`